packages/trader/src/AppV2/Containers/Positions/positions-content.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { TContractInfo } from '@deriv/shared';
+import { Loading } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { EmptyPositions, TEmptyPositionsProps } from 'AppV2/Components/EmptyPositions';
+import { TPortfolioPosition } from '@deriv/stores/types';
+import { ContractCardList, ContractCardsSections } from 'AppV2/Components/ContractCard';
+import { ContractTypeFilter, TimeFilter } from 'AppV2/Components/Filter';
+import TotalProfitLoss from 'AppV2/Components/TotalProfitLoss';
+import { filterPositions, getTotalPositionsProfit } from '../../Utils/positions-utils';
+import { TReportsStore, useReportsStore } from '../../../../../reports/src/Stores/useReportsStores';
+import useTradeTypeFilter from 'AppV2/Hooks/useTradeTypeFilter';
+import useTimeFilter from 'AppV2/Hooks/useTimeFilter';
+
+type TPositionsContentProps = Omit<TEmptyPositionsProps, 'noMatchesFound'> & {
+    hasButtonsDemo?: boolean;
+    setHasButtonsDemo?: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+export type TClosedPosition = {
+    contract_info: TReportsStore['profit_table']['data'][number];
+};
+
+const PositionsContent = observer(({ hasButtonsDemo, isClosedTab, setHasButtonsDemo }: TPositionsContentProps) => {
+    const { contractTypeFilter, setContractTypeFilter } = useTradeTypeFilter({ isClosedTab });
+    const { timeFilter, setTimeFilter, customTimeRangeFilter, setCustomTimeRangeFilter } = useTimeFilter();
+    const [filteredPositions, setFilteredPositions] = React.useState<(TPortfolioPosition | TClosedPosition)[]>([]);
+    const [noMatchesFound, setNoMatchesFound] = React.useState(false);
+
+    const { common, client, portfolio } = useStore();
+    const { server_time = undefined } = isClosedTab ? {} : common; // Server time is required only to update cards timers in Open positions
```

Comment:
We don’t want to pass `server_time` to Closed tab as it’s not used there and would cause unnecessary every-second re-renders. Default value is required, hence we set it to undefined according to its type accepted by children.